src/handler.js

```diff
@@ -68,7 +68,7 @@ module.exports = async (req, res) => {
     await cache.auth();
     const timingCacheAuth = Date.now() - timingCacheAuthStart;
 
-    let result;
+    let result = [];
```

Comment:
`result` is being used within the `finally` catch block,  which could break under some conditions. Adding an initial value helps preventing this